tests/README.md

```diff
@@ -0,0 +1,21 @@
+### Testing
+
+#### Requirements
+
+- OpenShift Container Platform v3.7
+- Permissions to use BuildConfig objects
```

Comment:
Maybe make it explicit that builds should happen on properly subscribed RHEL7 nodes?